signup-front/src/setupProxy.js

```diff
@@ -0,0 +1,9 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
I don't get the purpose of this 😅 . Can you 💡 me ?

Reply:
If you let the frontend access the back on a different port, you will enable the CORS mechanisms. Since there is no CORS in production, we want to avoir adding this ability in the backend only for the dev environment.

So the best solution is to use the CRA proxy feature :) https://create-react-app.dev/docs/proxying-api-requests-in-development/

The proxy can work with a single config in the `package.json`, but it will assume that the backend is accessible only through the `/api` route.

In order to extends this behaviour, we need to manually set the proxy.

Reply:
@Bernardstanislas = 🧞‍♂️